megamek/src/megamek/common/TurnOrdered.java

```diff
@@ -272,30 +272,56 @@ public static void rollInitiative(List<? extends ITurnOrdered> v, boolean bUseIn
         }
 
         rollInitAndResolveTies(v, null, bUseInitiativeCompensation);
+    }
+    
+    /**
+     * This takes a vector of TurnOrdered (Teams or Players), and does post
+     * initiative phase cleanup of the initiative streak bonus.
+     * 
+     * @param v
+     *            A vector of items that need to have turns.
+     * @param rerollRequests
+     * @param bInitCompBonus
+     *            A flag that determines whether initiative compensation bonus
+     *            should be used: used to prevent one side getting long init win
+     *            streaks
+     */
+    public static void resetInitiativeCompensation(List<? extends ITurnOrdered> v,
+            List<? extends ITurnOrdered> rerollRequests, boolean bInitCompBonus) {
```

Comment:
rerollRequests appears unused in this function. Let's get rid of it to keep the code clean.